### DISCUSSION TITLE
Documenting pixi and vscode integration

### DISCUSSION THREAD

**Discussion by lvjonok** [stack-of-tasks/pinocchio#2485] [Ideas]
Hello Pinocchio Team!

I am glad that this project is getting easier to build in predictive manner with `pixi` tool.

I wonder whether it will be appropriate to extend `development/build.md` with section about integration of this tool to widely used VSCode. I have searched for some time and found ongoing [issue](https://github.com/prefix-dev/pixi/issues/1570) by Silvio Traversaro. However there is not elegant, but working solution:

## Pixi setup for vscode

To conveniently use Pixi with Visual Studio Code for IntelliSense, you can configure [direnv](https://pixi.sh/latest/features/environment/#using-pixi-with-direnv).

Then, after moving to directory with **pinocchio** environment variables will be updated and running `code` from terminal will make IntelliSense work properly.

I hope that this note can be helpful for users and possible contributors.

**Comment by jorisv** [comment]
Hello @lvjonok,

You can create a PR to add these instructions and I think `development/build.md` is the right file to modify.
You can add a subsection under `# Build and install from source with Pixi.`

Also, did you try this VScode extension : https://github.com/direnv/direnv-vscode ?
I'm not a VScode user, so I don't know if this is useful.
